modules/nw-multus-host-device-object.adoc

```diff
@@ -1,30 +1,47 @@
 // Module included in the following assemblies:
 //
+// * networking/multiple-networks/configuring-host-device.adoc
 
 [id="nw-multus-host-device-object_{context}"]
 = Configuration for host-device
 
-// TODO - consider refactoring this into its own module as it is used
-// by both ipvlan and bridge CNI plug-ins.
+The configuration for an additional network attachment that uses the host-device
+Container Network Interface (CNI) plug-in is provided in two parts:
 
-.host-device CNI plug-in YAML configuration
+* Cluster Network Operator (CNO) configuration
+* CNI plug-in configuration
+
+The CNO configuration specifies the name for the additional network attachment
+and the Kubernetes namespace to create the attachment in. The plug-in
```

Comment:
Do we need _Kubernetes_ here? 
_and the namespace to create the attachment in_

Reply:
Perhaps not. Removed everywhere.